### PR TITLE
Update fachhochschule-sudwestfalen.csl

### PR DESCRIPTION
**Pull request by lukaslangede**
Changed et. al. disambiguate-add-names, and some formatting errors

### CODE REVIEW

File: fachhochschule-sudwestfalen.csl [hunk 1]
```diff
@@ -1,24 +1,22 @@
-<?xml version="1.0" encoding="utf-8"?>
-<style xmlns="http://purl.org/net/xbiblio/csl" class="note" version="1.0" et-al-min="4" et-al-use-first="1" demote-non-dropping-particle="sort-only" default-locale="de-DE">
+﻿<?xml version="1.0" encoding="utf-8"?>
+<style xmlns="http://purl.org/net/xbiblio/csl" class="note" version="1.0" et-al-use-first="1" et-al-subsequent-use-first="1" demote-non-dropping-particle="sort-only" default-locale="de-DE">
   <!-- This style was edited with the Visual CSL Editor (https://editor.citationstyles.org/visualEditor/) -->
   <info>
-    <title>Fachhochschule Südwestfalen (Deutsch)</title>
```
Comment (adam3smith) [inline fachhochschule-sudwestfalen.csl:5]: Please revert all.of these changes in info. The handling of Umlauts and other elements is standardized

File: fachhochschule-sudwestfalen.csl [hunk 2]
```diff
@@ -1,24 +1,22 @@
-<?xml version="1.0" encoding="utf-8"?>
-<style xmlns="http://purl.org/net/xbiblio/csl" class="note" version="1.0" et-al-min="4" et-al-use-first="1" demote-non-dropping-particle="sort-only" default-locale="de-DE">
```
Comment (adam3smith) [inline fachhochschule-sudwestfalen.csl:2]: et-al-min="1" doesn't make sense. What is the goal here in terms of output?

File: fachhochschule-sudwestfalen.csl [hunk 3]
```diff
@@ -119,7 +123,7 @@
       <if is-numeric="edition">
         <group delimiter=" ">
           <number variable="edition" form="ordinal"/>
-          <text term="edition" form="short" suffix=","/>
+          <text term="edition" form="short" strip-periods="true" suffix=".,"/>
```
Comment (adam3smith) [inline fachhochschule-sudwestfalen.csl:126]: Revert please

File: fachhochschule-sudwestfalen.csl [hunk 4]
```diff
@@ -88,14 +86,20 @@
   </macro>
   <macro name="year-date">
     <choose>
-      <if variable="issued">
-        <date variable="issued">
-          <date-part name="year"/>
-        </date>
+      <if match="none" position="ibid">
```
Comment (adam3smith) [inline fachhochschule-sudwestfalen.csl:89]: This doesn't look right: the year-date macro is used in the Bibliography, where the test doesn't apply

**Review by adam3smith — CHANGES_REQUESTED** [review]
Thanks.. Some questions, some requested fixes

File: fachhochschule-sudwestfalen.csl (hunk 1 above)
Comment (lukaslangede) [reply fachhochschule-sudwestfalen.csl:5]: thanks for the head up, I'm pretty new to this. I made the changes to the original file, so things should look better in the recent pull request

File: fachhochschule-sudwestfalen.csl (hunk 4 above)
Comment (lukaslangede) [reply fachhochschule-sudwestfalen.csl:89]: my bad - I think I fixed it now.

File: fachhochschule-sudwestfalen.csl (hunk 3 above)
Comment (lukaslangede) [reply fachhochschule-sudwestfalen.csl:126]: fixed now

**Comment by lukaslangede** [comment]
I made a new pull request with all changes. Sorry for this, I'm still figuring out github.

**Pull request closed without merging**